### PR TITLE
stable/satisfy: updating image to latest, with fixes

### PR DESCRIPTION
**Pull request by anapsix**
#### What this PR does / why we need it:
Updating to latest Docker image, which contains fixes for downloadable `package.json`

#### Checklist
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by anapsix** [comment]
oh well

**Comment by anapsix** [comment]
ping @cpanato, please approve

**Comment by anapsix** [comment]
ping @lachie83, could I please get an ok-to-test and further approvals

File: stable/satisfy/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: satisfy
-version: 0.1.1
+version: 0.1.2
 appVersion: "3.0.4"
```
Comment (cpanato) [inline stable/satisfy/Chart.yaml:3]: @anapsix please update the app version

Comment (anapsix) [reply stable/satisfy/Chart.yaml:3]: app version remained the same, changed were made to Nginx config of Docker image

Comment (anapsix) [reply stable/satisfy/Chart.yaml:3]: how do you suggest to proceed @cpanato, when docker image for application is updated, but application itself is not?

**Review by cpanato — APPROVED** [review]

**Pull request merged**